app/scripts/services/authorization.js

```diff
@@ -96,23 +98,37 @@ angular.module("openshiftConsole")
       return _.get(cachedRulesByProject.get(projectName || currentProject), ['rules']);
     };
 
-    var canI = function(verb, resource, projectName) {
-      projectName = projectName || currentProject;
-      var rules = getRulesForProject(projectName);
+    // _canI checks whether any rule allows the specified verb (directly or via a wildcard verb) on the literal group and resource.
+    var _canI = function(rules, verb, group, resource) {
+        var resources = rules[group];
+        if (!resources) { 
+          return false;
+        }
+        var verbs = resources[resource];
+        if (!verbs) { 
+          return false;
+        }
+        return _.contains(verbs, verb) || _.contains(verbs, '*');
+    };
+
+    // canI checks whether any rule allows the specified verb on the specified group-resource (directly or via a wildcard rule).
+    var canI = function(resource, verb, group, projectName) {
```

Comment:
you don't need a separate param for the group... resource should either be a string (which implies group='') or an object with `resource` and `group` fields